### PR TITLE
feat(docx-template): support inline images for docx-template

### PR DESCRIPTION
**Pull request by open-dynaMIX**
This commit implements inline images for docx-templates. This adds a new
`image` jinja filter. When passed a image filename, it will
include the provided file with the same name in the document as inline image.

The `merge` serializer now accepts multipart requests with an optional list of
image files.

Normal json requests without images still work as expected.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -231,6 +231,33 @@ Additionally, you can also convert output to pdf or other types supported by uno
 5031
 ```
 
+## Inline images
+
+The `docx-template` engine supports including inline images. Here is shown how one can
```
Comment (winged) [inline USAGE.md:236]: The local filename may not be the same as the one used in the template. Maybe add some documentation on how to "simulate" the filename that's transmitted, or just don't use the shortest way possible (use the syntax from request's example here https://requests.readthedocs.io/en/master/user/quickstart/#post-a-multipart-encoded-file )

File: document_merge_service/api/tests/test_template.py [hunk 2]
```diff
@@ -464,35 +465,82 @@ def test_template_merge_jinja_extensions_docx(db, client, template, settings, sn
     snapshot.assert_match(xml)
 
 
+@pytest.fixture
+def temp_png():
+    file_path = "/tmp/python-powered.png"
```
Comment (winged) [inline document_merge_service/api/tests/test_template.py:470]: Suggestion: use the `tmp_path` or `tmpdir` fixtures. They will also cleanup after themselves, so you don't have to

**Review by winged — CHANGES_REQUESTED** [review]
Some small suggestions, otherwise looks great!

File: USAGE.md (hunk 1 above)
Comment (open-dynaMIX) [reply USAGE.md:236]: Done!

File: document_merge_service/api/tests/test_template.py (hunk 2 above)
Comment (open-dynaMIX) [reply document_merge_service/api/tests/test_template.py]: Forgot about `tmp_path`, thanks! Done.

**Comment by open-dynaMIX** [comment]
@winged Tests fixed.

**Review by winged — APPROVED** [review]

**Pull request merged**